<ui version="4.0" >
 <class>NeuronProperty</class>
 <widget class="QWidget" name="NeuronProperty" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Neuron Parameters</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Please select a Neuron</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Statistics</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>N° neurons</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblNumNeurons" >
              <property name="text" >
               <string>0</string>
              </property>
              <property name="textFormat" >
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>N° inputs</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblNumInputs" >
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>N° weights</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lblNumWeights" >
              <property name="text" >
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Activation Function</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Steepness</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinSteepness" >
              <property name="buttonSymbols" >
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Function</string>
              </property>
              <property name="buddy" >
               <cstring>cmbActivationFunction</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="cmbActivationFunction" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Train Parameter</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Training Algorithm</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="cmbTrainAlgorithm" />
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedTrainParam" >
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <widget class="QWidget" name="page_3" >
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_8" >
                  <property name="text" >
                   <string>Learning Momentum</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinLearningMomentum" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_9" >
                  <property name="text" >
                   <string>Learning Rate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinLearningRate" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4" >
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_10" >
                  <property name="text" >
                   <string>Increase Factor</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinIncreaseFactor" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_11" >
                  <property name="text" >
                   <string>Decrease Factor</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinDecreaseFactor" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_12" >
                  <property name="text" >
                   <string>Delta Min</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinDeltaMin" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_13" >
                  <property name="text" >
                   <string>Delta Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinDeltaMax" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_14" >
                  <property name="text" >
                   <string>Delta Zero</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="spinDeltaZero" >
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::PlusMinus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>264</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
